File: back-tussam/src/main/kotlin/com/backtussam/model/Player.kt
package com.backtussam.model

import com.backtussam.db.tables.PlayerTable.default
import com.backtussam.db.tables.PlayerTable.integer

data class Player(
    val id: Int,
    val name: String,
    val lastName: String,
    val email: String,
    val location: String? = null,
    val nickname: String? = null,
    val avatar: String = "",
    var points: Int = 0,
    val createdAt: String,
    val active: Boolean = false,
    val gameWon: Int = 0,
    val gameLost : Int = 0,
    val gameDifference: Int = 0,
    val gamePlayed: Int = 0,
    val leagueId: Int? = null,
    val roleId: Int? = null,
    var authToken: String? = null
)

File: front-tussam/src/app/interfaces/Player.ts


export interface Player {
  id: number;
  name: string;
  lastName: string;
  email: string;
  userName?: string;
  avatar?: string;
  points?: number;
  createdAt: string;
  leagueId?: number | null;
  roleId?: number | null;
  authToken?: string | null;
}
